index.js

```diff
@@ -378,14 +377,14 @@ function evaluate(feature, user) {
     }
   }  
 
-  for (var i = 0; i < feature.variations.length; i ++) {
+  for (i = 0; i < feature.variations.length; i ++) {
```

Comment:
minor: why did you remove `var` here? technically I think that the declaration of `var i` from line 374 gets hoisted up, but it's not clear.